pdns/dnsdistdist/doh.cc

```diff
@@ -333,6 +333,16 @@ try
   h2o_socket_getsockname(sock, reinterpret_cast<struct sockaddr*>(&local));
   DOHServerConfig* dsc = reinterpret_cast<DOHServerConfig*>(req->conn->ctx->storage.entries[0].data);
 
+  // I wanted to put this near the other h2o_set_header method call, but should I move it?
+  for (auto const& headerPair : dsc->df->d_customResponseHeaders) {
+    const h2o_token_t *token = h2o_lookup_token(headerPair.first.c_str(), headerPair.first.size());
+    // TODO: Should I provide so WARN logging when we can't find a matching token?
+    if (token != NULL) {
```

Comment:
Just a nit but we try to use `nullptr`, which is not silently converted to a numerical value, for example.
```suggestion
    if (token != nullptr) {
```